### ISSUE TITLE
Create OpenCL-enabled Docker Node AMI & successfully run Sibernetic in a container on top of it

### ISSUE THREAD

**Issue by slarson** [openworm/sibernetic#105]
@skhayrulin and I experimented with running CPU-based sibernetic on Docker via AWS -- the result was a six day run for 1000 steps.  While it was helpful to run more than one simulation at a time (we ran five in parallel) it was overall still too slow.

What was good from this experiment was that @skhayrulin was able to specify the parameters to run in a short text file (like the one defined [here](https://github.com/openworm/sibernetic/issues/90#issuecomment-185053016)) and could easily spawn multiple runs with different parameters.  What was less successful about this strategy is that if we tried to distribute the simulations on more than a single node, the initial strategy of having a single central data node where we could upload results to dropbox from fell apart because for some reason the nodes were not able to communicate amongst each other.

The enabling technology for this was previously called Tutum, now it is called "Docker cloud".

@jamiemori and I met about a month back to discuss a strategy where we could run Sibernetic within our [OpenWorm docker container](hub.docker.com/r/openworm/openworm_test) via an AWS cluster that has GPUs.  While Docker cloud allows you to launch an AWS node that has GPUs, it doesn't come pre-configured with OpenCL; and it isn't sufficient to have OpenCL only installed in the Docker container; it needs to be installed on the node as well.  But we can address that problem separately in another issue.

For now, we want to speed up runs on AWS.

Now [recently](https://github.com/openworm/OpenWorm/issues/177#issuecomment-221728171), @tessbianchi has created [an EC2 image (ami-815dabec)](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#LaunchInstanceWizard:ami=ami-815dabec) that could be a candidate for this node.  @jamiemori's extra instructions to get the docker container working are [here](https://gist.github.com/jamiemori/4d437587f82acb50a459).

This issue is completed when we can use Docker Cloud to run Sibernetic within the docker container up on AWS on a GPU-enabled node and demonstrate a performance improvement over the CPU-enabled version.

Steps to accomplish this:
- [ ] Create a docker cloud / docker hub account if you don't have one
- [ ] Create an AWS account if you don't have one
- [ ] Launch the ami-815dabec from @tessbianchi as a g2 machine
- [ ] SSH into the node and use the instructions from docker cloud to [bring your own node](https://docs.docker.com/docker-cloud/infrastructure/byoh/).  This will enable the GPU-based host to run other docker containers within it.
- [ ] Within Docker cloud, create a new service from the [OpenWorm docker container](hub.docker.com/r/openworm/openworm_test), using the environment variables set up as defined by @jamiemori [here](https://gist.github.com/jamiemori/4d437587f82acb50a459).  You probably want to run the service in [privileged mode](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities), as suggested by @jamiemori's notes .  There may be some creativity required here as the paths may be somewhat different or there may be some missing or different libraries.  As an alternative to using the Docker Cloud GUI for configuring a service, a starting point may be to reuse part of the stackfile here (the `environment` section is a one to one match with setting local environment variables on the target container):

```
sibernetic-1:
 image: 'openworm/openworm_test:latest'
 entrypoint: /entrypoint.sh
 environment:
   - BRANCH_SIBERNETIC=development
   - CONF=worm_deep_water
   - OCL_FILE=sphFluid_crawling.cl
   - REBUILD_SIBERNETIC=true
   - SIBERNETIC_NAME=sibernetic-1
   - TIMELIMIT=2
```
- [ ] Report in this issue 3 or so steps of the log output generated from Sibernetic so we can see if the speed up is actually happening.
